coffee-backend/src/main/kotlin/chat/letscoffee/friends/FriendRequestController.kt

```diff
@@ -0,0 +1,49 @@
+package chat.letscoffee.friends
+
+import chat.letscoffee.exception.ResourceNotFoundException
+import chat.letscoffee.security.model.User
+import chat.letscoffee.security.repository.UserRepository
+import chat.letscoffee.security.security.CurrentUser
+import chat.letscoffee.security.security.UserPrincipal
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/friendrequest")
```

Comment:
I suppose we want to have a way to list friends in the future, so we could nest this hierarchically with a `/friends/requests` mapping, where `/friends` is for fetching confirmed friends.